### ISSUE TITLE
bash: clang.exe command not found 

### ISSUE THREAD

**Issue by lochoang75** [justmao945/vim-clang#129]
I used linux mint with clang 6.0 but i has this problem. how can i fix it ? Thank you.

**Comment by justmao945** [comment]
use clang. clang.exe is for Windows

**Comment by lochoang75** [comment]
yes, I use vim on linux terminal, but when i open vim it happen. I have installed and tested some file with my clang.

**Comment by deltaxflux** [comment]
You have to install clang.
